docs/make.jl

```diff
@@ -25,7 +32,7 @@ makedocs(
         assets = ["assets/wider.css", "assets/redlinks.css"],
         mathengine = Documenter.MathJax3(Dict(
             :tex => Dict(
-                "macros" => make_macros_dict("docs/src/assets/definitions.tex"),
+                "macros" => make_macros_dict("src/assets/definitions.tex"),
```

Comment:
```suggestion
                "macros" => make_macros_dict("docs/src/assets/definitions.tex"),
```

Should fix the CI error. Did you need this to build locally?